Explicitly label the `useAddressSelection` button

dimar01123 opened the pull request:
When adding a new location, it’s not very clear what clicking the `USE THIS LOCATION` button will do.

dimar01123 committed:
Explicitly label the `useAddressSelection` button

When adding a new location, it’s not very clear what clicking the `USE THIS LOCATION` button will do.

webprofusion-chrisc commented:
Hi, thanks for your suggested change! Unfortunately this would make the text far too long for some of our supported devices. I do appreciate you going to the effort of suggesting a change however in this case I don't agree that it's entirely necessary, Perhaps we could use a different choice of icon to indicate that the details will be transferred to the form? We're using the https://ionicons.com/ icon set.

dimar01123 committed:
An explicit short label on [useAddressSelection]

When adding a new location, it’s not very clear what clicking the [USE THIS LOCATION] button will do.

dimar01123 commented:
I’ve tried to be less verbose; does “Use this address below” fit and sound ok to you ?

webprofusion-chrisc commented:
Sounds good!
